### ISSUE TITLE
Supporting secure clusters from Windows clients

### ISSUE THREAD

**Issue by wesm** [ibis-project/ibis#475]
Getting the `kerberos` module to install on Windows is tricky. We should at minimum document the way forward for Windows users or come up with an alternate GSSAPI strategy

**Comment by nonsleepr** [comment]
I just finished (successfully) my attempts on installing `ibis-framework` on my Windows desktop.

`requests-kerberos` package used by `hdfs[kerberos]` uses `kerberos-sspi` on Windows (although wheel wrongly contains `any` in its name on PyPI, so it could only be installed with `pip install requests-kerberos --no-use-wheel`). **ibis** doesn't uses any `kerberos` packages explicitly, so there probably no need to force users have this packages.

Also, the project contains `snakebite` dependency, which is never used, and this one depends on `python-krbV`, also unavailable on Windows.

**Comment by wesm** [comment]
Thanks — will get this stuff cleaned up soon and put out in the next release (within a week or so)

**Comment by laserson** [comment]
Also note #480

**Comment by wesm** [comment]
@nonsleepr please let me know if you can contribute documentation on secure cluster configuration on Windows. Moving this to 0.5 milestone

**Comment by nonsleepr** [comment]
@wesm I have to figure out things with to CCLA in CS before starting doing that, so, not earlier that tomorrow.

**Comment by wesm** [comment]
no problem, thank you

**Comment by wesm** [comment]
hi @nonsleepr any updates on CCLA from your end? would love to have you involved.

**Comment by cpcloud** [comment]
@nonsleepr Closing this as the cleanups suggested here have all been done. Please open a new issue if there's still a problem.
